constants/domain_groups.json

```diff
@@ -1,724 +1,722 @@
-        "verbmobil",
-        "verbosity",
-        "videos",
-        "wayne magnuson english idioms sayings & slang",
-        "yoruba proverbs"
-    ],
-    "Reviews": [
-        "factcheck",
-        "fullfact",
-        "goodreads.com",
-        "hyatt.com",
-        "movie reviews",
-        "nielsen-netratings.com",
-        "rottentomatoes.com",
-        "tripadvisor",
-        "yahoo review",
-        "yelp"
-    ],
-    "Search Queries": [
-        "Google Suggest API",
-        "bing search queries",
-        "google search",
-        "google search queries",
-        "search-engine(google) auto-complete",
-        "sql2text",
-        "web search queries",
-        "webqsp",
-        "yandex"
-    ],
-    "Social Media": [
-        "4forums.com",
-        "blog authorship corpus",
-        "blogger.com",
-        "blogs",
-        "chinese online forums",
-        "convinceme.net",
-        "discussion forums",
-        "facebook",
-        "financial blogs",
-        "flickr",
-        "forum posts",
-        "gab.com",
-        "globalvoices.org",
-        "huffpost.com",
-        "javabkoo.com",
-        "linkedin",
-        "livejournal",
-        "quora",
-        "reddit",
-        "social media",
-        "spinn3r blogs",
-        "stormfront",
-        "theonion.com",
-        "twitter",
-        "yahoo! answers"
-    ],
-    "Template Generated": [
-        "grammar-based",
-        "templates"
-    ],
-    "Unsure": [
-        "annotation from native speaker annotators",
-        "bapra.bg",
-        "bianet",
-        "bnc",
-        "callhome spanish",
-        "capes",
-        "cbt",
-        "cometomyhead",
-        "cr",
-        "cyc.com",
-        "dainik jugasankha",
-        "dgt",
-        "digikala.com",
-        "dihana",
-        "ecb",
-        "ehuhac",
-        "elhuyar",
-        "enron corporation",
-        "etv",
-        "eur-lex portal",
-        "frontpagemag.com",
-        "george lakoff’s conceptual metaphor list",
-        "inf.uni-hamburg.de/en/inst/ab/lt/resources/data/depcc.html",
-        "infopankki",
-        "infotabs.github.io",
-        "judge.u-aizu.ac.jp",
-        "kattis",
-        "kdedoc",
-        "komi",
-        "logic2text",
-        "loria",
-        "mbs",
-        "media.mit.edu/projects/",
-        "moniteur belge/belgisch staatsblad",
-        "montenegrinsubs",
-        "nahj al-balagha",
-        "ofispublik",
-        "openstax.org",
-        "php",
-        "pirt tests",
-        "pri",
-        "procon.org",
-        "prompts",
-        "pullum and huddleston 2002 for premises",
-        "quizziz;study.com;proprofs",
-        "restaurant information dialogs",
-        "rf",
-        "sadilar",
-        "sardware",
-        "sbic",
-        "sqa",
-        "tep",
-        "tiwall.com",
-        "ukwac",
-        "wenshu.court.gov.cn",
-        "xhosanavy",
-        "zhihu",
-        "zujuan.xkw.com"
-    ]
-}
\ No newline at end of file
+  "Academic Papers": [
```

Comment:
Could we re-format as it was before? It's hard to see if there were any changes. Thanks!